### DISCUSSION TITLE
Namecheap domain configuration invalid

### DISCUSSION THREAD

**Discussion by mikeborozdin** [vercel/community#5260] [Help]
### Summary

I've bought a domain name of Namecheap and I cannot assign it to my Vercel project. 

I had done it for another domain registered with Namecheap and I've used Vercel's DNS for that.

I've followed the same procedure and put `ns1.vercel-dns.com` and `ns2.vercel-dns.com` to the `nameservers` of my new domain. But Vercel still says that the configuration is invalid.

If I run `whois <my domain>` I do get:

```
Name Server: ns1.vercel-dns.com
Name Server: ns2.vercel-dns.com
```

### Steps to Reproduce

n/a

**Comment by amyegan** [comment]
Hey @mikeborozdin. If you changed the nameservers very recently, it could simply be a normal DNS propagation delay. It can take up to 24-48 hours for a nameserver change to take effect.

https://www.namecheap.com/support/knowledgebase/article.aspx/9622/10/dns-propagation-explained/

**Comment by mikeborozdin** [comment]
> Hey @mikeborozdin. If you changed the nameservers very recently, it could simply be a normal DNS propagation delay. It can take up to 24-48 hours for a nameserver change to take effect.
> 
> https://www.namecheap.com/support/knowledgebase/article.aspx/9622/10/dns-propagation-explained/

Hi @amyegan ,

Thank for the response! Why does the `whois` command then return new nameservers?

**Reply by amyegan** [reply]
If you share the domain, I can look into it further for you

**Comment by mikeborozdin** [comment]
All good! The problem was with the slow registration. I had never registered a country domain(.ai is a country domain) and everything was taking hours instead of usual seconds.
